libvast/src/system/evaluator.cpp

```diff
@@ -170,7 +171,7 @@ caf::behavior evaluator(caf::stateful_actor<evaluator_state>* self,
         auto& curried_pred = get<1>(triple);
         auto& indexer = get<2>(triple);
         st.predicate_hits[pos].first += 1;
-        self->request(indexer, caf::infinite, curried_pred)
+        self->request(indexer, defaults::system::request_timeout, curried_pred)
           .then([=](const ids& hits) { self->state.handle_result(pos, hits); },
                 [=](const caf::error& err) {
                   self->state.handle_missing_result(pos, err);
```

Comment:
Do we have to distinguish between timeouts and `missing result` errors here now?

Reply:
I'm currently implementing checks for the response handles. The CI failure still has me puzzled, and I want to find out if it is related to a request shutdown in some way, but I fail to reproduce it. (This answer applies to the next two comments as well.)

Reply:
I've come to the conclusion that we must keep using `infinite` for at least the evaluator.